Gemfile

```diff
@@ -284,7 +284,7 @@ gem 'unf_ext', '0.0.7.2'
 gem 'acmesmith', '~> 2.3.1'
 
 gem 'addressable'
-gem 'bcrypt'
+gem 'bcrypt', '3.1.13'
```

Comment:
this may not be necessary since the commit will point us to this PR, but we could add a comment here that explains 1) why we set bcrypt to this version, and 2) to make sure any version we upgrade to doesn't introduce a similar regression